src/test/java/seedu/weme/logic/commands/FindCommandTest.java

```diff
@@ -16,13 +16,14 @@
 import seedu.weme.model.ModelManager;
 import seedu.weme.model.UserPrefs;
 import seedu.weme.model.meme.TagContainsKeywordsPredicate;
+import seedu.weme.statistics.StatsManager;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code MemeFindCommand}.
  */
-public class MemeFindCommandTest {
-    private Model model = new ModelManager(getTypicalMemeBook(), new UserPrefs());
-    private Model expectedModel = new ModelManager(getTypicalMemeBook(), new UserPrefs());
+public class FindCommandTest {
+    private Model model = new ModelManager(getTypicalMemeBook(), new UserPrefs(), new StatsManager());
+    private Model expectedModel = new ModelManager(getTypicalMemeBook(), new UserPrefs(), new StatsManager());
```

Comment:
What's the reason for the rename?

Reply:
I guess for a period of time I used FindCommand for main app, then changed back. This test class name was changed to FindCommandTest but never got changed back.